src/filters/display-name-filter.js

```diff
@@ -0,0 +1,33 @@
+(function (angular, _) {
+    'use strict';
+
+    var drf = angular.module('drf-display-name-filter', []);
+
+    // A filter to return a display name.
+    drf.filter('drfDisplayName', function () {
+        return _.memoize(function (item, array) {
+            var displayName = item;
+
+            var foundName = _.find(array, function (arrayItem) {
+                if (item === arrayItem.value) {
+                    return arrayItem;
+                }
```

Comment:
Just `return item === arrayItem.value`

Actually I prefer `arrayItem.value === item` in terms of operand order